src/super_gradients/common/sg_loggers/base_sg_logger.py

```diff
@@ -329,6 +330,21 @@ def _save_checkpoint(self, path: str, state_dict: dict) -> None:
         if self.save_checkpoints_remote:
             self.model_checkpoints_data_interface.save_remote_checkpoints_file(self.experiment_name, self._local_dir, name)
 
+    @multi_process_safe
+    def add_yaml_summary(self, tag: str, summary_dict: dict, global_step: Optional[int] = None) -> None:
```

Comment:
Perhaps you should also add support for W&B & ClearML & DagsHub loggers to upload this file to their platform as well (See how add_checkpoint is implemented there)

Reply:
It seems, that the part of the code repeats in all those loggers + the logic of adding checkpoints repeats somehow, so I guess it should be refactored a bit. Please, let me do it in a separate pull request.